Lib/defcon/objects/glyph.py

```diff
@@ -161,6 +163,11 @@ def __del__(self):
         self._lib = None
         self._image = None
 
+    def postNotification(self, notification, data=None):
+        if self._isLoading:
+            return
+        super().postNotification(notification, data)
```

Comment:
that's python3-only, you need `super(ClassName, self).methodName(*args, **kwargs)`